File: TableIperation_WinForms/DGV_DataSource/Person.cs
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace DGV_DataSource
{
    public class Person
    {
        [DisplayName("Имя пользователя")]
        public string Name { get; set; }
        [DisplayName("Возраст")]
        public int Age { get; set; }
        [DisplayName("Рост")]
        public double Height { get; set; }
        [DisplayName("Вес")]
        public double Weight { get; set; }

        public Person(string name, int age, double height, double weight)
        {
            Name = name;
            Age = age;
            Height = height;
            Weight = weight;
        }
    }
}

File: Помелов М.А/TS/Lesson 2/person.ts
class Person {
    name: string;
    age: number;
    height: number;
    phoneNumber: string;
}

let person: Person = new Person();
person.name = 'Valery';
person.age = 34;
person.height = 1.78;
person.phoneNumber = '891231231231';

console.log(`
    Имя: ${person.name}\n
    Возраст: ${person.age}\n
    Рост: ${person.height}\n
    Номер: ${person.phoneNumber}\n
    `);